### PR TITLE
Remove deprecated method AssertionAdapter::getAttribute()

### PR DESCRIPTION
**Pull request by DRvanR**

### CODE REVIEW

**Comment by DRvanR** [comment]
@rjkip @arothuis

File: src/SAML2/Response/AssertionAdapter.php [hunk 1]
```diff
@@ -65,20 +60,6 @@ public function getNameID()
...
-     */
-    public function getAttribute($name, $default = null)
-    {
-        return $this->getAttributeValue($name, $default);
-    }
-
-    /**
      * @param string $attributeName
      * @param null   $defaultValue
      * @return null|string[]
```
Comment (rjkip) [inline src/SAML2/Response/AssertionAdapter.php]: While you're at it: `$defaultValue`'s type should be `mixed|null`. The return value must consequently be changed to `@return string[]|mixed`. Or maybe we should require `$defaultValue` to be an array of strings.

**Comment by DRvanR** [comment]
@rjkip @arothuis

**Comment by rjkip** [comment]
:sparkles::rocket::sparkles:

**Comment by arothuis** [comment]
:ship:

**Pull request merged**